Automattic/wp-calypso
# Untangling: Add Advanced Tools -> Logs

Fixes:

- https://github.com/Automattic/dotcom-forge/issues/9618

## Proposed Changes

This PR adds the Advanced Tools -> Monitoring subtab under the `untangling/hosting-menu` flag.

<img width="1125" alt="image" src="https://github.com/user-attachments/assets/54229c5a-6f50-4311-8e10-1594eb8438d1">


## Why are these changes being made?

pbxlJb-6ye-p2

## Testing Instructions

1. Go to /sites
2. Click an Atomic site
3. Go to Advanced Tools -> Logs subtab.
4. Verify that this tool is working (check for php and web subtabs).
5. Regression testing:
    - Go to /sites?flags=-untangling/hosting-menu
    - Click an Atomic site
    - Verify that the original Logs tab is still working.

## Pre-merge Checklist
